CI.md

```diff
@@ -32,19 +32,26 @@ libyaml        v0.2.5
 - `--with-yaml`
 - `--enable-test-input=/home/unit_tests_input`
 
+### Build libFMS with autotools (intel)
+
+Required Intel build test for all pull requests.
+Runs `make check` after configuring via autotools.
+
+This runs on an intel-hosted container image from dockerhub.
+
+To access the netcdf and libyaml the dependencies, it builds and caches the resulting libaries for reuse.
```

Comment:
`the` seems out of place here.

Reply:
The second "the".
To access the netcdf and libyaml **the** dependencies

Reply:
yes, *that* `the`